test/url-to-options.js

```diff
@@ -0,0 +1,56 @@
+import url from 'url';
+import test from 'ava';
+import urlToOptions from '../source/url-to-options';
+
+const githubURL = 'https://alex:pass@github.com:443/';
+
+test('Converts node legacy URL to options', t => {
+	const parsedURL = url.parse(githubURL);
+	const options = urlToOptions(parsedURL);
+	const expected = {
+		hash: null,
+		hostname: 'github.com',
+		href: 'https://alex:pass@github.com:443/',
```

Comment:
`href: githubURL`,

Reply:
Why didn't you suggest this approach for the second test 🤔 ? Did you know it would break?